scRNA-seq/02-normalizing_scRNA-seq.Rmd

```diff
@@ -67,185 +66,298 @@ sc_metadata <- readr::read_tsv(metadata_file)
 ## Set up a SingleCellExperiment object from count data
 
 Now that we have filtered our data, we are ready to normalize it.
-To do this, we are going to use some single-cell RNA-seq R packages called `scater` and `scran`.
+To do this, we are going to use some single-cell RNA-seq R packages called `scater` and `scran`. 
 
 ![](diagrams/full-length_2.png)
 
-The functions in these packages require the data to be in their own special 
-object type (not an uncommon thing for R packages to do) called `SingleCellExperiment`.
+The functions in these packages require the data to be in their own special object type (not an uncommon thing for R packages to do) called `SingleCellExperiment`.
 So first, we are going to set up our data in this format.
-To learn more about `SingleCellExperiment` objects and how it works with
-Bioconductor packages, we recommend this article: [Amezquita et al. 2020.](https://www.nature.com/articles/s41592-019-0654-x).
-
-```{r Set up SCE object}
-# Prep a matrix with just the rounded counts so we can create an sce object
-counts_mat <- round(as.matrix(sc_data_filtered[, -1]))
+To learn more about `SingleCellExperiment` objects and how it works with Bioconductor packages, we recommend this article: [Amezquita *et al.* 2020.](https://www.nature.com/articles/s41592-019-0654-x).
 
-```
-
-Put our counts matrix in this special `SingleCellExperiment` function: 
 
 ```{r}
 # Set up the Single Cell Experiment object 
-sce_raw <- SingleCellExperiment::SingleCellExperiment(list(counts = counts_mat))
-
-# save some memory
-rm(counts_mat)
+gbm_sce <- SingleCellExperiment(list(counts = as.matrix(sc_data_filtered[, -1])))
 
 # Store the gene names in this object
-rowData(sce_raw) <-  sc_data_filtered$gene
+rownames(gbm_sce) <-  sc_data_filtered$gene
 ```
 
-In the intro-to-R-tidyverse module notebook, `01-intro-to-r`, we discuss base R 
-object types, but there are some 'special' object types that are 
-package-specific. 
-`SingleCellExperiment` objects, for example, are what a lot of single-cell 
-analysis R packages use, so we will try to get acquainted with them. 
+In the intro-to-R-tidyverse module notebook, `01-intro-to-r`, we discuss base R object types, but there are some 'special' object types that are package-specific. 
```

Comment:
```suggestion
In the intro-to-R-tidyverse module notebook, [`01-intro-to-base_R.Rmd`](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) we discuss base R object types, but there are some 'special' object types that are package-specific. 
```

Reply:
I don't love linking this because it may break for updated versions, and I can't think of a way to properly handle that.

Reply:
Meh. Delete it then. But probably make sure the notebook name is accurate. aka keep the `_base_R.Rmd` part.

Reply:
yup, kept that!